setup.py

```diff
@@ -203,6 +204,51 @@ def run(self):
             sys.exit(128)
 
 
+class Test(SimpleCommand):
+    """Run selftests"""
+
+    description = 'Run selftests'
+    user_options = [
+        ("job-api", None, "Run job API checks"),
+        ("static-checks", None, "Run static checks (isort, lint, etc)"),
+        ("nrunner-interface", None, "Run selftests/functional/test_nrunner_interface.py"),
```

Comment:
nrunner-interface is a specific type of "functional". While I understand the importance of this test, I'm wondering what was the criteria to create a specific target for it here.

My concern is that in the future this list will be huge because every single functional/unit test will have an own user-option. What do you think?

Reply:
@beraldoleal I was following the logic we have in `check.py`. For some reason, there were a group of test name nrunner-interface and I followed that.

Reply:
I see @ana. 

@clebergnu, what do you think about my point? Do you have any objections on removing this?

Reply:
@beraldoleal I'm opening an issue with this, it risks to get buried here with so many things.

Reply:
That is fine @ana. We can discuss this there or here, however IMO, the outcome of this discussion it is important for this PR.

Reply:
Btw, probably you have notice, but I'm marking some threads here as "resolved" to get less "buried".